config/environments/development.rb

```diff
@@ -242,6 +242,21 @@
   # This variable sets the number of old log files that will be kept
   config.x.logging.old_files = 10
 
+  ###################################################################
+  # Email Notifications
+  ###################################################################
+  config.action_mailer.delivery_method = :smtp
```

Comment:
Make sure all of these new settings are in `test.rb` and `production.rb` as well.